### PR TITLE
Bugfix for #35

### PR DESCRIPTION
**Pull request by rgostic**
Added trim and trailing ellipsis for items longer than 15 characters. Prevents cart quantity vertical alignment.

### CODE REVIEW

**Comment by atharva3010** [comment]
Looks good! I have some code which will introduce merge conflicts, so let me clean it up and then we'll merge your PR.

File: src/components/checkout/CartItem.vue [hunk 1]
```diff
@@ -50,14 +50,22 @@ export default {
...
   methods: {
     increment() {
       this.$store.dispatch('cart/incrementCartProduct', this.cartItem.product);
     },
     decrement() {
       this.$store.dispatch('cart/decrementCartProduct', this.cartItem.product);
     }
+  },
+  computed: {
+    cartItemTitle() {
```
Comment (atharva3010) [inline src/components/checkout/CartItem.vue:63]: Making this a method and then passing title as argument in that method would be more usable, what do you think?

Comment (rgostic) [reply src/components/checkout/CartItem.vue:63]: It depends, if the font size is different then the method "shouldn't" work for both font sizes.

Comment (rgostic) [reply src/components/checkout/CartItem.vue:63]: Unless I make it a method that takes a character count argument.

Comment (rgostic) [reply src/components/checkout/CartItem.vue:63]: But we could utilize that everywhere really.

Comment (atharva3010) [reply src/components/checkout/CartItem.vue:63]: Exactly, let's make it more reusable! We can keep this method in this component only for now, if we come across another similar use-case, we will make it into a filter

**Pull request merged**